pkg/helm/helm.go

```diff
@@ -328,15 +328,15 @@ func upgradeRelease(ctx context.Context, local client.Client, restClientGetter g
 }
 
 func installRelease(ctx context.Context, local client.Client, restClientGetter genericclioptions.RESTClientGetter, plugin *greenhousev1alpha1.Plugin, pluginConfig *greenhousev1alpha1.PluginConfig, isDryRun bool) (*release.Release, error) {
-	cfg, err := newHelmAction(restClientGetter, pluginConfig.Namespace)
+	cfg, err := newHelmAction(restClientGetter, pluginConfig.GetReleaseNamespace())
 	if err != nil {
 		return nil, err
 	}
 	installAction := action.NewInstall(cfg)
 	installAction.ReleaseName = pluginConfig.Name
-	installAction.Namespace = pluginConfig.Namespace
-	// Namespaces are only created by an Organization.
-	installAction.CreateNamespace = false
+	installAction.Namespace = pluginConfig.GetReleaseNamespace()
+	// Create the target namespace if it doesn't exist.
+	installAction.CreateNamespace = true
```

Comment:
Note the change here. 
The logic adheres to the Helm behavior and creates the namespace, but doesn't delete it. 
Only the Helm release is managed by Greenhouse to natively integrate into existing setups.